packages/java/src/browser/java-client-contribution.ts

```diff
@@ -44,9 +48,25 @@ export class JavaClientContribution extends BaseLanguageClientContribution {
 
     protected onReady(languageClient: ILanguageClient): void {
         languageClient.onNotification(ActionableNotification.type, this.showActionableMessage.bind(this));
+        languageClient.onNotification(StatusNotification.type, this.showStatusMessage.bind(this));
         super.onReady(languageClient);
     }
 
+    protected showStatusMessage(message: StatusReport) {
```

Comment:
@JPinkney, please reset the timeout in the beginning:

```
        if (this.statusBarTimeout) {
            window.clearTimeout(this.statusBarTimeout);
            this.statusBarTimeout = undefined;
        }
```

Reply:
Added!

Reply:
@JPinkney, thanks! Looks good. Waiting for CI...